tests/transforms/test_Rotation.py

```diff
@@ -1,14 +1,30 @@
 from PIL import Image
 import unittest
 
-from lightly.transforms import RandomRotate
+from lightly.transforms.rotation import RandomRotate, RandomRotateDegrees, random_rotation_transform
 
 
-class TestGaussianBlur(unittest.TestCase):
+class TestRotation(unittest.TestCase):
 
-    def test_on_pil_image(self):
+    def test_RandomRotate_on_pil_image(self):
         for w in range(1, 100):
             for h in range(1, 100):
                 random_rotate = RandomRotate()
                 sample = Image.new('RGB', (w, h))
                 random_rotate(sample)
+    
+    def test_RandomRotateDegrees_on_pil_image(self):
+        for w in range(1, 10):
+            for h in range(1, 10):
+                for r in range(1, 90):
```

Comment:
This will run 90 tests. We don't need that many as it will slow down the test suite a lot, I would propose something like this, then we also cover the tuple input types:
```
for degrees in [0, 1, 45, (0, 0), (-15, 30)]:
```